### PR TITLE
Change active menu item for Fraud protection

### LINKED ISSUE #6369: Clicking on Fraud Protection menu item in wpadmin jumps to another menu
### Describe the bug
Currently, when you click on the `Payments > Fraud protection` menu item in a Woo Express site, it opens up the `Extensions` menu. With the default menu structure, the same menu item opens up the `WooCommerce` menu.

### Screenshots

Here is a GIF showing the behavior on the Woo Express site.

wooexpress-paymentsfraudprotection-jumps-to-extensions

And here is the same behavior with the default WP menu structure found in the issue linked below.

240589396-11fcc1c6-e05b-460c-841e-8f909906b442

### Additional context

This was investigated briefly in Automattic/wc-calypso-bridge#1159, where they found the issue was not a result of the ecommerce menu controller.

### PR DESCRIPTION
**Pull request by tpaksu**
Fixes #6369

### Changes proposed in this Pull Request

This PR changes the active menu item shown for Fraud Protection screen from ___WooCommerce > Settings___ to ___Payments > Settings___ page, which causes problems on WooExpress as the WooCommerce menu is moved to ___Settings > WooCommerce___ and the ___WooCommerce > Settings___ menu is rendered under the ___Extensions___ menu. 

##### Not onboarded:
* Navigate to `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ffraud-protection` and you should see **Sorry, you are not allowed to access this page.**
* Check if any **Fraud Protection** menu exists under both _WooCommerce_ or _Payments_ menus. Shouldn't exist.

##### Onboarded
* Navigate to `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ffraud-protection` and you should see the Advanced fraud protection settings page.
* Check if Payments > Settings is the active item while you're on that page.
* Check if any **Fraud Protection** menu exists under both _WooCommerce_ or _Payments_ menus. Shouldn't exist.
* Click the **WooCommerce Payments** link on the breadcrumb on the top of the form. It should redirect you to the **Payments > Settings** page.
* Click the **Configure** button on the Fraud protection box, first select the Advanced radio button to make it clickable. 
* You should see the Advanced fraud protection settings page, and **Payments > Settings** should be the active menu item.

##### Not onboarded:
* Navigate to `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ffraud-protection` and you should see the Advanced fraud protection settings page.
* Check if Payments is the active item while you're on that page.
* Check if any **Fraud Protection** menu exists under, _Extensions_, _Settings_, or _Payments_ menus. Shouldn't exist.
* Click the **WooCommerce Payments** link on the breadcrumb on the top of the form. It should redirect you to the **Settings > WooCommerce** page.
* Click the **Configure** button on the Fraud protection box, first select the Advanced radio button to make it clickable. 
* You should see the Advanced fraud protection settings page, and **Payments** should be the active menu item.

#### Onboarded: 
* Navigate to `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ffraud-protection` and you should see the Advanced fraud protection settings page.
* Check if **Payments > Settings** is the active item while you're on that page.
* Check if any **Fraud Protection** menu exists under, _Extensions_, _Settings_, or _Payments_ menus. Shouldn't exist.
* Click the **WooCommerce Payments** link on the breadcrumb on the top of the form. It should redirect you to the **Settings > WooCommerce** page.
* Click the **Configure** button on the Fraud protection box, first select the Advanced radio button to make it clickable. 
* You should see the Advanced fraud protection settings page, and **Payments > Settings** should be the active menu item.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6715` or branch name `fix/6369-wooexpress-remove-fraud-protection-menu-disconnected` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=fix/6369-wooexpress-remove-fraud-protection-menu-disconnected) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: fb664271b18d753f1012226a180f1ca8dfa74738
- Build time: 2023-07-10 09:54:21 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/settings/fraud-protection/advanced-settings/index.tsx [hunk 1]
```diff
@@ -188,14 +188,14 @@ const FraudProtectionAdvancedSettingsPage: React.FC = () => {
 	// Hack to make "WooCommerce > Settings" the active selected menu item.
```
Comment (PanosSynetos) [inline client/settings/fraud-protection/advanced-settings/index.tsx]: Hey @tpaksu , shouldn't this be "Payments > Settings" ?

**Review by PanosSynetos — APPROVED** [review]
Thanks @tpaksu for this fix!

Manually tested on a woo express site, and works as expected.

As for a code review, it seems good, but I'd prefer if someone with more experience on the repo to code review it too.

File: client/settings/fraud-protection/advanced-settings/index.tsx (hunk 1 above)
Comment (tpaksu) [reply client/settings/fraud-protection/advanced-settings/index.tsx]: True, forgot to fix the comment :) Thanks! Addressed in [fb66427](https://github.com/Automattic/woocommerce-payments/pull/6715/commits/fb664271b18d753f1012226a180f1ca8dfa74738).

**Review by allie500 — APPROVED** [review]
Thanks for fixing this bug, Taha! The WCPay tests passed for me. The code changes are small and easy to understand. LGTM! :shipit:

**Pull request merged**